Lib/ipaddress.py

```diff
@@ -1849,7 +1869,7 @@ class IPv6Address(_BaseV6, _BaseAddress):
 
     """Represent and manipulate single IPv6 Addresses."""
 
-    __slots__ = ('_ip', '__weakref__')
+    __slots__ = ('_ip', '_scope_id', '__weakref__')
```

Comment:
```suggestion
    __slots__ = ('_ip', 'scope_id', '__weakref__')
```
I believe the underscore before `scope_id` is unneeded here and in the `__init__()`. IIRC, when an instance attribute has a defined `@property`, `class.attr` will call the defined property method instead of directly accessing the attribute.